### PR TITLE
Removes bf-sim from the startup flags.

### PR DESCRIPTION
**Pull request by pierventre**
It is no longer supported.

### CODE REVIEW

**Review by charlesmcchan — APPROVED** [review]

**Review by pudelkoM — COMMENTED** [review]
You'll probably want to add `-enable_onlp=false` now, unless ONLP works flawlessly on the switches.

File: stratum/templates/configmap-startup.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ data:
     set -e -x
     LOCAL_CHASSIS_CONFIG=$CFG_MOUNT/chassis_config.pb.txt
     if [ -f $LOCAL_CHASSIS_CONFIG ]; then
-        /usr/bin/start-stratum.sh -bf-sim -chassis_config_file=$LOCAL_CHASSIS_CONFIG {{ template "extraParams" .Values.extraParams }}
+        /usr/bin/start-stratum.sh -enable_onlp=false -chassis_config_file=$LOCAL_CHASSIS_CONFIG {{ template "extraParams" .Values.extraParams }}
```
Comment (pudelkoM) [inline stratum/templates/configmap-startup.yaml:17]: Any reason not to put it into `extraParams` of `values.yaml`?

Comment (charlesmcchan) [reply stratum/templates/configmap-startup.yaml:17]: Is ONLP fully working?
If not, disabling it by default makes sense to me

Comment (pudelkoM) [reply stratum/templates/configmap-startup.yaml:17]: Usually it's not, disabling is fine. My comment was more about the location of the flag, not its existence.

Comment (charlesmcchan) [reply stratum/templates/configmap-startup.yaml:17]: I consider `extraParam` more like a way to supply optional paramters / override parameters. 
If this should be disable by default across all deployments, I have no issue leaving it outside of `extraParam`

**Review by pudelkoM — APPROVED** [review]

**Pull request merged**